style.css

```diff
@@ -1,4 +1,312 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   font-family: Montserrat, sans-serif;
+  margin: 0 auto;
+  width: 1440px;
+  padding: 0;
+}
+
+.header,
+.collection {
+  background: linear-gradient(to right, black 32%, white 10%);
+}
+
+.header {
+  align-items: center;
+  display: flex;
+  font-size: 0;
+  height: 92px;
+  width: 100%;
+}
+
+.header__logo {
+  display: flex;
+  height: 92px;
+  margin-right: 90px;
+  padding: 36px 52px 19px 52px;
+  width: 202px;
+}
+
+.language {
+  display: flex;
+  height: 92px;
+  padding: 46px 108px 55px 97px;
+  width: 259px;
+}
+
+.language__img {
+  height: 25px;
+  width: 31px;
+}
+
+.language__arrow {
+  height: 8px;
+  margin: 10px;
+  width: 16px;
+}
+
+.header-menu {
+  background-color: #f2f2f2;
+  height: 92px;
+  margin-left: 1348px;
+  position: absolute;
+  top: 0;
+  width: 92px;
+}
+
+.header-menu__line::before {
+  content: '';
+  transform: translateY(-11px);
+}
+
+.header-menu__line, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LOCO_proj_Boiko/pullRequest?prid=5658198)